docs/en/sql-reference/aggregate-functions/parametric-functions.md

```diff
@@ -506,3 +506,254 @@ Solution: Write in the GROUP BY query SearchPhrase HAVING uniqUpTo(4)(UserID) >=
 ## sumMapFiltered(keys_to_keep)(keys, values) {#summapfilteredkeys-to-keepkeys-values}
 
 Same behavior as [sumMap](../../sql-reference/aggregate-functions/reference/summap.md#agg_functions-summap) except that an array of keys is passed as a parameter. This can be especially useful when working with a high cardinality of keys.
+
+## sequenceNextNode {#sequenceNextNode}
+
+Returns a value of next event that matched an event chain.
+
+**Syntax**
+
+``` sql
+sequenceNextNode(direction, base)(timestamp, base_condition, event_column, event1, event2, event3, ...)
+```
+
+**Parameters**
+-   `direction` - Used to navigate to directions.
+    - forward : Moving forward
+    - backward: Moving backward
+
+-   `base` - Used to set the base point.
+    - head : Set the base point to the first event
+    - tail : Set the base point to the last event
+    - first_match : Set the base point to the first matched event1
+    - last_match : Set the base point to the last matched event1
+    
+**Arguments**
+-   `timestamp` — Name of the column containing the timestamp. Data types supported: `Date`, `DateTime` and other unsigned integer types.
+-   `base_condition` — Condition that the base point must fulfill.
```

Comment:
Why it can be first event from `cond` ?

Reply:
`base_condition` can be moved to the first event-argument with `and combinator`. 

i.e.
```SQL
sequenceNextNode('forward', 'first_match')(date_time, ref_sns = 'facebook', page, page = 'home')
```
The above query can be replaced with the below query.
```SQL
sequenceNextNode('forward', 'first_match')(date_time, page, page = 'home' and ref_sns = 'facebook')
```

I will remove the `base_condition`.

Reply:
`base_condition` can be moved to the first event-argument with `first_match` or `last_match`. 

i.e.
```SQL
sequenceNextNode('forward', 'first_match')(date_time, ref_sns = 'facebook', page, page = 'home')
```
The above query can be replaced with the below query.
```SQL
sequenceNextNode('forward', 'first_match')(date_time, page, page = 'home' and ref_sns = 'facebook')
```

But, `base_condition` is required with `head` and `tail` because there is no way to specify that condition.